components/atom/tag/src/index.scss

```diff
@@ -10,6 +11,7 @@ $p-atom-tag-l: 0 $p-l !default;
 $p-atom-tag-m: 0 $p-l !default;
 $p-atom-tag-s: 0 $p-m !default;
 $w-atom-tag-clickable: 32px !default;
+$mq-atom-tag-responsive-breakpoint: 's' !default;
```

Comment:
Did ux define that this should be for every breakpoint? I mean, this may break the coherence between other components. You should be using hardcoded 's'  so everyone uses its 's' breakpoint

Reply:
in which paper is this defined?

Reply:
You are right. I talk about it with Marc Garrido and moved 's' value to the media mixin